### ISSUE TITLE
Einthusan does not work with KODI v18 Leia

### ISSUE THREAD

**Issue by onyx64** [humla/canadanepal-xbmc#34]
I have tried this on multiple platforms. The videos keep skipping after every few seconds and this happens only on the KODI Leia (v18). v17.6 is fine.

**Comment by san6031** [comment]
Yes. I have the same issue with v18 . It was working fine with v17

**Comment by hemantkapoor** [comment]
Hello, 

I am having the same issue.
Can someone please have a look.... Many thanks for all the support.

**Comment by eintamil** [comment]
Use EinTamil Repo: https://eintamil.github.io/
Video Addon Einthusan2 for Kodi 18.x (Python 2.x)
Video Addon Einthusan3 for Kodi 19.x (Python 3.x)
If any issues, raise at https://github.com/eintamil/repository.eintamil/issues
